test/check-translation-test.js

```diff
@@ -157,7 +157,7 @@ exports.check_translations = {
 
   'malformed html - no opening or closing tag name': function (test) {
     checkTranslation(
-      '<>no opening tag name</>',
+      '<>no opening or closing tag name</>',
```

Comment:
Does this actually test the `</>` component of the string?  ISTM that it should always fail on the leading `<>` and we should have a separate test for just the `</>`.